examples/with-mongodb/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "with-mongodb",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "mongodb": "^3.5.7",
+    "mongoose": "^5.9.13",
+    "next": "^9.4.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+  },
+  "description": "Next.js + MongoDB Example",
+  "main": "next.config.js",
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "dotenv": "^8.2.0"
+  }
```

```suggestion
  "license": "ISC"
```

No dotenv dep, please.